src/basic/log.h

```diff
@@ -165,7 +165,7 @@ int log_format_iovec(
                 bool newline_separator,
                 int error,
                 const char *format,
-                va_list ap);
+                va_list ap)  _printf_(6,0);
```

Comment:
How can GCC check the argument type inside the va_list ? This is really supported ?